boards/holybro/durandal-v1/nuttx-config/bootloader/defconfig

```diff
@@ -92,7 +91,6 @@ CONFIG_TIME_EXTENDED=y
 CONFIG_TTY_SIGINT=y
 CONFIG_TTY_SIGINT_CHAR=0x03
 CONFIG_TTY_SIGSTP=y
-CONFIG_USART3_DMA=y
```

Comment:
Did this go in on purpose?

Reply:
Looks like something forgotten in https://github.com/PX4/Firmware/pull/14771/files

Reply:
Leftovers that disappeared automatically after running menuconfig/qconfig.